seismic_hazard_analysis/utils.py

```diff
@@ -168,3 +168,58 @@ def is_pd(B: np.ndarray[float]):
         return True
     except sp.linalg.LinAlgError:
         return False
+
+
+def get_min_max_levels_for_im(im: str):
```

Comment:
Some of this might need to change if dealing with new IM outputs due to changed units for CAV etc.
Might be completely fine but good to double check

Reply:
This is for hazard, doesn't have anything to with IM calc? Or are you just meaning for consistency?

Reply:
I guess for consistency, wasnt sure if this function was going to use IM_calc outputs to generate hazard either